tests/integration/integration-test.sh

```diff
@@ -168,8 +175,27 @@ load_test() { #
   rm output.log
 }
 
+### CREATE IMAGEPULLSECRET ####################################
+create_imagepullsecret() { # NAMESPACE # CREATE
+  local CREATE=${2:-true}
+  if $CREATE; then
+    echo -n "Creating Namespace '${1}'..."
+    kubectl create ns ${1}
```

Comment:
Creation of a NS is a weird and unintuitive consequence of creating a secret

Reply:
yes, but cannot use an imagePullSecret from another NS and wanted to avoid code duplication or unintuitively adding a namespace all the time. What do you propose?

Reply:
Maybe rename to something that at least hints at the dependency on ns, like `create_imagepullsecret_in_ns`

Otherwise would just do the creation and fail if ns not present and add ns in respective locations explicitly (and if unintuitive, add comment)

Reply:
OK, renamed it to be more explicit since it is tests (not production code) and code duplication is more painful than explicitness is valuable in tests. hope that is fine